components/NavigationBar/tests/unit/NavigationBarTests.m

```diff
@@ -199,6 +200,32 @@ - (void)testTitleFontProperty {
   XCTAssertEqual(fontTraits, resultTraits);
 }
 
+- (void)testTitleFontPropertyWithAllowAnyTitleFontSizeEnabled {
+  MDCNavigationBar *navBar = [[MDCNavigationBar alloc] init];
+  navBar.frame = CGRectMake(0, 0, 300, 25);
+  navBar.title = @"this is a Title";
+  navBar.titleAlignment = MDCNavigationBarTitleAlignmentCenter;
+  navBar.allowAnyTitleFontSize = YES;
+  [navBar layoutIfNeeded];
+
+  XCTAssertNotNil(navBar.titleFont);
+  XCTAssertEqual(navBar.titleLabel.font, navBar.titleFont);
```

Comment:
Please consider using `XCTAssertEqualObjects`.  Should this be the exact same object or equal objects? Is it acceptable if the UILabel were to copy the font we provided?

Reply:
Done.